tests/test_utils.py

```diff
@@ -0,0 +1,33 @@
+import pytest
```

Comment:
no I think it should be under test_redirect.py since all it does is tests redirects work properly (which I am glad you added a test for)